src/components/data-processing/CalculationConfigContainer.jsx

```diff
@@ -0,0 +1,111 @@
+import React, { useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import PropTypes from 'prop-types';
+import {
+  Space,
+  Form,
+  Button,
+  Alert,
+  Radio,
+} from 'antd';
+
+import _ from 'lodash';
+
+import { updateProcessingSettings } from '../../redux/actions/experimentSettings';
+
+const CalculationConfigContainer = (props) => {
```

Comment:
can you write some unit tests for this component?

Reply:
This has been surprisingly painful, and, unfortunately, I cannot say now that I have a much better understanding of how react testing works than before starting :_( 

And I still had to comment out on of the tests :-(

Reply:
I can see that :(

Reply:
Tests are painful, I know, sometimes takes more time to write the tests than to write the functionality.